scripts/evaluate_single_receipt.py

```
@@ -0,0 +1,263 @@
+                line_item_patterns=patterns,
+                chroma_client=chroma_client,
+                dynamo_client=dynamo_client,
+            )
+            logger.info(f"  Reviewed {len(reviewed_issues)} issues")
+        finally:
+            # Cleanup ChromaDB client
+            if chroma_client is not None:
+                try:
+                    chroma_client.close()
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)